users/migrations/0001_initial.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 4.0.3 on 2022-03-24 18:53
+
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
```

Comment:
if the verbose_name is the same or the same as the default maybe we can get rid of it
```suggestion
                ('password', models.CharField(max_length=128)),
```
same for all the rest occurrences in this PR

Reply:
The file '0001_initial.py' is a computer generated file. 
When we change the variable name than the next time we run 'make migrations' it creates another file to correct the variable name.